test/client/js/libs/handlebars/ircmessageparser/parseStyle.js

```diff
@@ -104,12 +109,70 @@ describe("parseStyle", () => {
 		expect(actual).to.deep.equal(expected);
 	});
 
+	it("should parse hex colors", () => {
+		const input = "test \x04FFFFFFnice \x02\x04RES006 \x0303,04\x04ff00FFcolored";
```

Comment:
Hmm, AdiIRC seems to allow foreground and background. I should make it more clear on the doc that it allows setting both foreground and background colours with hex in this way